### PR TITLE
[MODULAR][MDB IGNORE]Re-adds the (renamed) Victorian Vest and Dutch Assistant costumes

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Had some people voice sadness over losing these; managed to actually make some good improvements to them
- Victorian Vest (now `buttondown shirt with double-breasted vest`)[Something less wordy? Just double-breasted vest implies it's worn in the suit slot...] is now fully Greyscale, down to the buttons! It uses the Buttondown Shirt/Slacks as a base to make it consistent with those and save me time 😎 
- Dutch Assistant (now `cavalry uniform`)[This took a while to research, especially bc despite the name/icon state, Arthur Morgan has nothing to do with it] now has the suspenders not vanish at the shoulders on the front, better coloring/shading, and all dirs are even! [The pants on the sides used to be like 2-3px shorter how the hell]

I would have marked this as no gbp but these sprites actually took a bit of effort to clean up

## How This Contributes To The Skyrat Roleplay Experience
The people have spoken, and I am a benevolent ~~god~~ **sprite cleaner**

Plus damn they actually look p good
dreamseeker_1jOW9TrBAQ
dreamseeker_AsNKfPjAMw
image

image

## Changelog

:cl:
add: (Re)Added the totally new and unique "Cavalry Uniform" (originally Dutch Assistant) and "Buttondown Shirt with Double-breasted Vest" (originally Victorian Vest)
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
ok the defines don't work :( I'll deal with it 2morrow

File: modular_skyrat/master_files/code/modules/clothing/under/costume.dm [hunk 1]
```diff
@@ -15,6 +15,23 @@
 *	UNSORTED
 */
 
+//This item's icons are in shorts_pants_shirts.dmis! Why? Because TG has obj/item/clothing/under/costume/buttondown's icons in there and this reuses some of those.
+//Ifever TG decides to either move the buttondown to its own type or move the icons to costume.dmi, we'll follow their example.
+/obj/item/clothing/under/costume/skyrat/vicvest //there's no way I'm typing out a path called double_breasted 10 times over, too complex and everyone will be scared of it
+	name = "buttondown shirt with double-breasted vest"
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/costume.dm:21]:
```suggestion
	name = "buttondown shirt with vest"
```
Is the double-breasted part important?

Comment (ghost) [reply modular_skyrat/master_files/code/modules/clothing/under/costume.dm:21]: Somewhat? Its how its distinguished from TGs suit-vests

File: modular_skyrat/master_files/code/modules/clothing/under/costume.dm [hunk 2]
```diff
@@ -15,6 +15,23 @@
...
+	desc = "A fancy buttondown shirt with slacks and a vest worn overtop, with a second row of buttons. Truly an outdated fashion statement."
+	icon_state = "buttondown_vicvest"
+	greyscale_config = /datum/greyscale_config/buttondown_vicvest
+	greyscale_config_worn = /datum/greyscale_config/buttondown_vicvest/worn
+	greyscale_config_worn_digi = /datum/greyscale_config/buttondown_vicvest/worn/digi
+	greyscale_colors = "#8b2c2c#222227#222227#fbc056"
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/under/costume/skyrat/arthur
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/costume.dm]:
```suggestion
/obj/item/clothing/under/costume/skyrat/cavalry
```
Arthur is super unclear what it is and makes it difficult to locate

**Comment by GoldenAlpharex** [comment]
We're four tomorrows later 😢

File: modular_skyrat/master_files/code/modules/clothing/under/costume.dm (hunk 2 above)
Comment (ghost) [reply modular_skyrat/master_files/code/modules/clothing/under/costume.dm]: valid, I'll get 'em all changed. honestly spent a while wondering wtf the uniform even was so it slipped my mind to just CALL it what it was

**Comment by ghost** [comment]
> We're four tomorrows later 😢

I knowwwww I'm sorryyyy I was at my gma's funeral and then I was just tired and sleepy I'm a sleeper a dozer a snoozin' boy.

I'm just gonna do the Defines in a different PR because this one hates me and it's meant to be re-adding these items for people who asked and holding it up to add defines feels bleh

File: modular_skyrat/modules/GAGS/json_configs/costumes/misc.json [hunk 3]
```diff
@@ -6,5 +6,31 @@
 			"blend_mode": "overlay",
 			"color_ids": [ 1 ]
 		}
+	],
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/json_configs/costumes/misc.json]: TIL, the gags .jsons scream if you have two states with different amounts of color_ids. I'll need to make a unique .json for this damn vest.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**